src/Data/Impact.elm

```diff
@@ -71,6 +73,15 @@ type alias PefData =
     }
 
 
+type alias Scope =
+    List Target
```

Comment:
I think you can directly have a `Scope` type:

```elm
type Scope = Food | Textile
```

And a `scopes` property (note the plural form) which would be a `List Scope`.